packages/ember-testing/lib/helpers/click.js

```diff
@@ -10,14 +10,14 @@ import { focus, fireEvent } from '../events';
   Example:
 
   ```javascript
-  click('.some-jQuery-selector').then(function() {
+  click('.some-css-selector').then(function() {
     // assert something
   });
   ```
 
   @method click
-  @param {String} selector jQuery selector for finding element on the DOM
-  @param {Object} context A DOM Element, Document, or jQuery to use as context
+  @param {String} selector css selector for finding element on the DOM
```

Comment:
Although the functionality has not changed and still uses jquery when it's available, this docs change could be misleading, suggesting there is a breaking change. Shouldn't this state something like `jQuery selector or css selector (if jQuery is not present) for finding element on the DOM`?

Reply:
Oh, seems you changed it already while I wrote this!? :)

Reply:
That seems like a good option.  I have it now as `@param {String|Element} target the element or selector to click on`.  What do you think of this?  Should it be:

`the element or jQuery selector or css selector (if jQuery is not preset) to click on`

Reply:
Seems good to me